src/test/java/com/amazon/opendistroforelasticsearch/security/auditlog/compliance/ComplianceAuditlogTest.java

```diff
@@ -86,6 +93,36 @@ public void testSourceFilter() throws Exception {
         Assert.assertTrue(validateMsgs(TestAuditlogImpl.messages));
     }
 
+    @Test
+    public void testComplianceEnable() throws Exception {
+        Settings additionalSettings = Settings.builder()
+                .put("opendistro_security.audit.type", TestAuditlogImpl.class.getName())
+                .build();
+
+        setup(additionalSettings);
+        rh.sendAdminCertificate = true;
```

Comment:
Order in which tests are executed by JUnit is not defined, this may be the first one to be executed and it will set `sendAdminCertificate` to `true`.